sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/implementation/FutureTemperatureParameters.java

```diff
@@ -9,11 +9,16 @@
 import com.azure.json.JsonWriter;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
 
 public class FutureTemperatureParameters implements JsonSerializable<FutureTemperatureParameters> {
 
     private String type = "object";
 
+    private Boolean additionalProperties = false;
```

Comment:
It is from this property in openai-api:
https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L9472

If we don't provide this property, the response will fail with missing `additionalProperties` property.

Reply:
dang it. That's a weird imposition on customer models for this sort of stuff. It would be cool if the backend assumed a default value of `false` here if it can't be nullable. 

I will bring this up in our meeting today and discuss whether it would be a good idea to inject the value ourselves if not present.